src/app/components/pages/books/Workbook20AQA.tsx

```diff
@@ -1,21 +0,0 @@
-import {Col, Container} from "reactstrap";
```

Comment:
Just a quick question: how come it's OK to remove this file (and the other work book)? It looks like they were being used in RoutesCS.tsx.

Reply:
As above, they were placeholder pages. They weren't linked anywhere on the website and if you manually navigated to the page using the route, it didn't load the content because it was unpublished - it looks like this was something being done 3 years ago that was never finished. 

The information all still exists and is live at https://isaaccomputerscience.org/pages/ocr_workbook_2022 and https://isaaccomputerscience.org/pages/aqa_workbook_2022, which is what you find when going to Learn->Workbooks (https://isaaccomputerscience.org/pages/workbooks_2020) via main menu.